README.rst

```diff
@@ -55,6 +63,8 @@ Required arguments:
 
 ``batch_scoring --host=<host> --user=<user> <project_id> <model_id> <dataset_filepath> --datarobot_key=<datarobot_key> {--password=<pwd> | --api_token=<api_token>}``
 
+``batch_scoring_sse --host=<host> <import_id> <dataset_filepath>``
+
 Additional recommended arguments
```

Comment:
It seems that this displays like this:

Additional recommended arguments [--verbose] [--keep_cols=<keep_cols>] [--n_concurrent=<n_concurrent>]

Additional optional arguments [--out=<filepath>] [--api_version=<api_version>] [--pred_name=<string>] [--timeout=<timeout>] [—-create_api_token] [--n_retry=<n_retry>] [--delimiter=<delimiter>] [--resume] [--skip_row_id] [--output_delimiter=<delimiter>]

It should display like this:

Additional recommended arguments:

 [--verbose] [--keep_cols=<keep_cols>] [--n_concurrent=<n_concurrent>]

Additional optional arguments:

 [--out=<filepath>] [--api_version=<api_version>] [--pred_name=<string>] [--timeout=<timeout>] [—-create_api_token] [--n_retry=<n_retry>] [--delimiter=<delimiter>] [--resume] [--skip_row_id] [--output_delimiter=<delimiter>]